src/game/Globals/ObjectMgr.h

```diff
@@ -721,7 +721,8 @@ class ObjectMgr
         CreatureAddon const* GetCreatureTemplateAddon(uint32 entry);
         ItemTemplate const* GetItemTemplate(uint32 entry);
         ItemTemplateContainer const* GetItemTemplateStore() const { return &_itemTemplateStore; }
-
+        std::vector<ItemTemplate*> const* GetItemTemplateStoreFast() const { return &_itemTemplateStoreFast; }
+    
```

Comment:
can you please remove those whitespaces?

Reply:
Which do you mean? I just took the line above it as a reference

Reply:
@Kaev https://i.imgur.com/kAOGNT5.png trailing whitespace (whitespace at the end of line).

Reply:
already done :)